pkgs/applications/networking/mailreaders/thunderbird/wrapper.nix

```diff
@@ -26,7 +26,7 @@ browser: args:
     buildCommand =
       old.buildCommand
       + ''
-        wrapProgram $out/bin/${browser.binaryName} \
+        wrapProgram "$executablePath" \
```

Comment:
Can we use the nix variable instead of bash?

Reply:
This one contains `$out`. Regardless, we'll have to maintain the same shell variable inside the Firefox wrapper buildCommand, so it's probably fine to use it here too. Otherwise we'd have to pass the nix variable calculated in the original wrapper code here, and that's not very obvious.